homeassistant/components/utility_meter/sensor.py

```diff
@@ -187,6 +184,7 @@ def __init__(
         self._sensor_net_consumption = net_consumption
         self._tariff = tariff
         self._tariff_entity = tariff_entity
+        self._attr_unique_id = f"utility_meter{parent_meter}{tariff}"
```

Comment:
Maybe split the unique_id to make it more readable
```suggestion
        self._attr_unique_id = f"utility_meter_{parent_meter}_{tariff}"
```